fluidefi/fluidefi-caspernet-aggregator-tools
# Block hours parser

**Summary of change:**
- Added a new micro-service, block hours parser, which accepts block fields as an argument and returns the parsed block. 

**List of tasks:** 


**Steps for testing**
- Make sure your local git is up to date
```bash
git pull
```

- Checkout the branch `feature/block-hours-parser`
```bash
git checkout feature/block-hours-parser
```

- Run the test across the test-net
```bash
npm run test
```

- Confirm block-parser test has passed.
- Run example.
